main.js

```diff
@@ -0,0 +1,22 @@
+const menu = document.querySelector('.menu');
+const menuOption = document.querySelector('.menu_opt');
+const closeMenu = document.getElementById('exit');
+const menuItems = document.querySelector('.menu_list');
+
+function show() {
+  menuOption.style.display = 'block';
+  menuOption.classList.toggle('open');
+  menu.style.visibility = 'hidden';
+  // closeMenu.style.visibility ='visible';
```

Comment:
- [ ] [optional] It's best practice to keep your code clean. So I recommend removing unnecessary commented lines of code